pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -209,10 +209,14 @@ func ValidateValidatingWebhookConfiguration(e *admissionregistration.ValidatingW
 }
 
 func validateValidatingWebhookConfiguration(e *admissionregistration.ValidatingWebhookConfiguration, opts validationOptions) field.ErrorList {
+	needValidateLabelValue := false
+	if e != nil && e.UID == "" {
+		needValidateLabelValue = true
```

Comment:
this should be added to `validationOptions` as a boolean... to compute this properly, we need the old and new ValidatingWebhookConfiguration, not just to check if uid is non-empty

Reply:
same comments apply everywhere... add the "validate label selector label values" option to existing validation options structs, and compute based on the old object's selectors